assets/css/combined.css

```
@@ -1,4 +1,930 @@
+    margin: 0;
+    position: relative;
+  }
+  #gheader .header-col-right .navi-list.menu li:first-child {
+    margin-left: 0;
+  }
+  #gheader .header-col-right .navi-list.menu li a {
+    display: block;
+    letter-spacing: 0.02em;
+    height: 80px;
+    display: flex;
+    align-items: center;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)